lib/gtl_formatter.rb

```diff
@@ -125,10 +125,13 @@ def self.format_cols(view, row, controller, options)
                 :text  => text}.compact
       elsif COLUMN_WITH_TIME.include?(col)
         celltext = format_time_for_display(row, col)
-      elsif COLUMN_WITH_OS_ICON.key?(col)
-        @osicon = send(COLUMN_WITH_OS_ICON[col], record)
-        @ostext = format_col_for_display(view, row, col).presence || _("Unknown")
       elsif COLUMN_WITH_OS_TEXT.include?(col)
+        view.col_order.each do |column|
```

Comment:
I'm not sure I understand why we have to walk all of the columns again when we are already walking them above on line 94.

Reply:
Image is coming from different column  [Here](https://github.com/ManageIQ/manageiq-ui-classic/blob/70f9415205c3887b76ed53ae4bb1c958cb82e192/product/views/VmOrTemplate.yaml#L68)  and text is coming from different one [Here](https://github.com/ManageIQ/manageiq-ui-classic/blob/70f9415205c3887b76ed53ae4bb1c958cb82e192/product/views/VmOrTemplate.yaml#L57). not sure if there is another way..

Reply:
Oh that's weeeeeird...need to think about this.

Reply:
Are osicon and ostext used anywhere else outside of this code?  If not we can make them local variables and not instance variables.